backend/src/main/java/rocks/inspectit/gepard/agentmanager/connection/model/dto/CreateConnectionRequest.java

```diff
@@ -4,27 +4,29 @@
 import jakarta.validation.constraints.NotNull;
 import java.time.Instant;
 import java.util.Map;
-import java.util.UUID;
 import rocks.inspectit.gepard.agentmanager.agent.model.Agent;
 import rocks.inspectit.gepard.agentmanager.connection.model.Connection;
+import rocks.inspectit.gepard.agentmanager.connection.model.ConnectionStatus;
 
 /** Represents a connection request from an agent. */
 public record CreateConnectionRequest(
     @NotNull(message = "Service Name missing.") String serviceName,
     @NotNull(message = "Gepard Version missing.") String gepardVersion,
-    @NotNull(message = "Open-Telemetry Version missing.") String otelVersion,
-    @NotNull(message = "Process ID is missing.") Long pid,
+    @NotNull(message = "OpenTelemetry Version missing.") String otelVersion,
+    @NotNull(message = "VM-ID is missing.") String vmId,
+    @NotNull(message = "Agent-ID is missing") String agentId,
     @NotNull(message = "Start-Time missing.") Long startTime,
     @NotNull(message = "Java Version missing.") String javaVersion,
     Map<String, String> attributes) {
 
   public static Connection toConnection(CreateConnectionRequest createConnectionRequest) {
     return new Connection(
-        UUID.randomUUID(),
```

Comment:
Is it that bad, if both, the connection and the agent have their own id´s?

Reply:
I don't think it makes sense to exaggerate with the IDs. We already have one value, which identifies the agent as well as the connection. Since it's a 1:1 relation, why would we create another value, which identifies the exact object